cps/tasks/download.py

```diff
@@ -95,6 +95,12 @@ def run(self, worker_thread):
                             if error:
                                 log.error("[xklb] An error occurred while trying to download %s: %s", error[1], error[0])
                                 self.message = f"{error[1]} failed to download: {error[0]}"
+                            else:
+                                log.error("No error found in the database, likely the video failed due to unavailable fragments.")
```

Comment:
The goal should be very complete diagnostic hints in both... logs and in "Tasks" view error messages:

@deldesir: what log.error message is best?

```suggestion
                                log.error("%s failed to download: No path or error found in the database (likely the video failed due to unavailable fragments?)", self.media_url_link)
```

Reply:
Best, but the use of self.media_url_link here is not appropriate in a log message. Use self.media_url instead.

Reply:
Sounds good, go ahead.

Reply:
Thanks @deldesir: please enact `self.media_url` if that's best, and test!